### PR TITLE
Fixed Geo/Stat/DCAT-AP entries

### PR DESCRIPTION
**Pull request by andrea-perego**
- Added aliases for major and latest versions
- Replaced original URLs with the corresponding permanent URIs

### CODE REVIEW

File: refs/biblio.json [hunk 1]
```diff
@@ -3239,67 +3239,71 @@
             }
         }
     },
+
     "dcat-ap": {
-        "href": "https://joinup.ec.europa.eu/asset/dcat_application_profile/",
+        "href": "http://data.europa.eu/w21/ac376c94-74cf-4dd7-ade7-267d6a4ec4dc",
```
Comment (marcoscaceres) [inline refs/biblio.json]: No https?

Comment (tobie) [reply refs/biblio.json]: @andrea-peregocan you please address this comment? Thanks!

Comment (tobie) [reply refs/biblio.json]: @andrea-perego can you please address this comment? Thanks!

Comment (andrea-perego) [reply refs/biblio.json]: @marcoscaceres , @tobie , these persistent URIs are operated by the Persistent URI Registry of EU Institutions and Bodies (http://data.europa.eu/), which currently uses HTTP. 

I will update them once HTTPS will be supported.

Comment (marcoscaceres) [reply refs/biblio.json]: Thanks for checking.

**Pull request merged**